src/couch/src/couch_debug.erl

```diff
@@ -18,6 +18,18 @@
     opened_files_contains/1
 ]).
 
+-export([
+    process_name/1,
+    link_tree/1,
+    link_tree/2,
+    mapfold/3,
+    map/2,
+    fold/3,
+    linked_processes_info/2,
+    print_linked_processes/1,
+    ps/1
```

Comment:
The idea is to add more clauses to display information specific to couchdb components:
- couch_index_server
- couch_server
- couch_event_server
I implemented the `couch_index_server` case only because I am currently debugging its behavior. 

I was thinking to eventually support multiple orders of results:
- hierarchical (by links in order of Pid values)
- hierarchical (by monitors in order of Pid values)
- hierarchical based on supervision tree 
- order by value in on of the columns 

However these features are outside of the scope of the current PR.

I agree with you that `ps` is not the best name for it.